cwms-http-client/src/test/java/mil/army/usace/hec/cwms/http/client/TestCwmsHttpLogger.java

```diff
@@ -0,0 +1,83 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.logging.Level;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestExecutor;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.jupiter.api.Test;
+
+final class TestCwmsHttpLogger {
+
+    private static final String ACCEPT_HEADER_V1 = "application/json";
```

Comment:
Instead of setting a StringBuilder on the log interceptor, use a log handler like this:

```
AtomicBoolean foundMessage = new AtomicBoolean(false);
Logger.getLogger("okhttp3").addHandler(new Handler() {
    @Override
    public void publish(LogRecord record) {
        String message = record.getMessage();
        if(message.contains(body)) {
            foundMessage.set(true);
        }
    }

    @Override
    public void flush() {

    }

    @Override
    public void close() throws SecurityException {

    }
});
```